static/js/containers/CourseIndexPage.js

```diff
@@ -85,6 +92,11 @@ export const CourseIndexPage = ({
     <Grid className="main-content one-column">
       {loaded ? (
         <Cell width={12}>
+          <CourseCarousel
```

Comment:
Maybe just display this if there are any favorites, like featured courses below?
```javascript
{favorites.length !== 0 ? (.....) : null}
```

Reply:
in the designs there's sort of an "empty" state for the favorites bar, so even if the user doesn't have anything favorited we want to show something. I didn't implement the specific display for this yet just because the mockups weren't finalized and approved and everything yet

Reply:
OK, never mind this then, LGTM.  I just introduced a conflict though by merging the programs/paths PR - should those lists now be expected to show up in the Favorites carousel too?

Reply:
yeah it should show all favorites